examples/demo-rollup/README.md

```diff
@@ -74,78 +74,254 @@ circumstances.
+01000000d40000007cb06da843cb98a223cdd4aee61ea4533f99104fe03144720d75800580d9a665be112c73b8d0b02b8de73f678d2432e93f613071e6fd04cc96b6ab5e6952bf007b758bf2e7670fafaf6bf0015ce0ff5aa802306fc7e3f45762853ffc37180fe66800000001000b000000000000000e000000736f762d746573742d746f6b656ee803000000000000a3201954f70ad62230dc3d840a5bf767702c04869e85ab3eee0b962857ba759801000000a3201954f70ad62230dc3d840a5bf767702c04869e85ab3eee0b962857ba75980100000000000000
+```
+* The output can be redirected to a file so that we can use it with the `make` command from earlier
+```
+user@machine sovereign % ./target/debug/sov-cli make-blob ./examples/demo-stf/src/sov-cli/test_data/create_token.dat > ./examples/demo-stf/src/sov-cli/test_data/celestia_blob
+```
+* To submit the blob, we'll start from scratch (since the test transaction we submitted has the same nonce, token fields etc)
+```
+cd examples/demo-rollup
+make clean
+make start
+```
+* Start the demo-rollup
+```
+cd examples/demo-rollup
+cargo +nightly run
+```
+* Submit the transaction
+```
+user@machine sovereign % cd examples/demo-rollup
+user@machine demo-rollup % SERIALIZED_BLOB_PATH=../demo-stf/src/sov-cli/test_data/celestia_blob make submit-txn
 ```
 
-- `xxd` is used to convert the serialized file into hex to post as an argument to `celestia-appd`
-- `736f762d74657374` is the namespace `ROLLUP_NAMESPACE` in `examples/demo-rollup/src/main.rs`
-- `01000000b000000004ee8ca2....` is the serialized binary blob in hex
-- `sequencer_keypair` is the keypair created earlier and should also match the value of `SEQUENCER_DA_ADDRESS` in `examples/demo-rollup/src/main.rs`
-- `celestia-appd` asks for confirmation - accept with y/Y
 
 ### Verify the supply of the new token created
 
 ```
-$ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"bank_supplyOf","params":["sov1jzvd95rjx7xpcdun2h8kyqee2z5r988h3wy4gsdn6ukc5ae04dvsrad3jj"],"id":1}' http://127.0.0.1:12345
-{"jsonrpc":"2.0","result":{"amount":5000},"id":1}
+$ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"bank_supplyOf","params":["sov16m8fxq0x5wc5aw75fx9rus2p7g2l22zf4re72c3m058g77cdjemsavg2ft"],"id":1}' http://127.0.0.1:12345
+{"jsonrpc":"2.0","result":{"amount":1000},"id":1}
 ```
 
-- params: should be the token address created in step 6
+### Remote setup
+The above setup runs celestia node locally to avoid any external network dependencies and to speed up development. The sovereign sdk can also be configured to 
+connect to the celestia testnet using a celestia light node running on your machine. T
+here are instructions on how to do this at [Remote Setup](remote_setup.md) the remote setup has a dependency on the versions of the testnet, the light client as well as the adapters use to connect to the light client.
+Currently, the remote setup doesn't work due to breaking changes but the general process is still the same if developers wish to try different versions for the nodes
```

Comment:
```suggestion
At present, the remote setup is not functional because the Celestia testnet version that our Celestia adapter supports has been sunsetted. We are collaborating with the Celestia team to update the adapter.
```